executor/src/witgen/generator.rs

```diff
@@ -115,18 +136,25 @@ impl<'a, T: FieldElement> Generator<'a, T> {
 
     fn process<Q: QueryCallback<T>>(
         &self,
-        first_row: Row<'a, T>,
+        data: Vec<Row<'a, T>>,
```

Comment:
This is quite confusing because there is `self.data` and `data` - what is the difference?